src/interfaces/modular/some.i

```diff
@@ -0,0 +1,15 @@
+%{
+    #include <shogun/base/some.h>
+    #include <shogun/base/SGObject.h>
+    #include <shogun/base/Parallel.h>
+    #include <shogun/io/SGIO.h>
+%}
+
+%rename (ptr) operator CSGObject*;
+%rename (ptrSGIO) operator SGIO*;
+%rename (ptrParallel) operator Parallel*;
+%include <shogun/base/some.h>
+
+%template (SomeSGObject) shogun::Some<CSGObject>;
```

Comment:
this is a good test, but i'm thinking that we dont actually wanna do this (see my email to you @geektoni) but want to actually for example instead of 
```
%rename(GaussianShiftKernel) CGaussianShiftKernel;
```
in `Kernel.i` we would only expose the `Some<>` version:
```
%template (GaussianShiftKernel) shogun::Some<CGaussianShiftKernel>;
```